src/js/utils/__tests__/RouterUtil-test.js

```diff
@@ -68,27 +73,32 @@ describe("RouterUtil", function() {
       ]);
       const component = components[0].props.children;
 
-      expect(
-        ReactTestUtils.isElementOfType(component, ReactRouter.Redirect)
-      ).toBeTruthy();
+      try {
+        // Duck typing to check if it is a route
```

Comment:
I know, this is not the nicest solution, but I found it easier and better then other similar soltions that would have removed ReactTestUtils

Reply:
Let's try another approach 🙇

Reply:
Let's pair on this, I have no idea how to do it differently